src/test/ui/feature-gate-try_reserve.stderr

```diff
@@ -0,0 +1,11 @@
+error[E0658]: the `#[try_reserve]` attribute is an experimental feature
+  --> $DIR/feature-gate-try_reserve.rs:13:1
+   |
+LL | #[try_reserve] //~ ERROR: attribute is an experimental feature
```

Comment:
This should also reflect the error gotten from a unstable library feature.

Reply:
Thanks, solved!